hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/validators/TokenAttributesValidator.java

```diff
@@ -159,4 +191,18 @@ public static boolean isKeyRemoval(@NonNull final Key source) {
         requireNonNull(source);
         return IMMUTABILITY_SENTINEL_KEY.equals(source);
     }
+    /**
+     * Checks if the given Bytes contains a zero
+     * @param bytes the Bytes to check
+     * @return true Bytes contains zero, false otherwise
+     */
+    public boolean containsZeroByte(@NonNull final Bytes bytes) {
+        byte[] byteArray = bytes.toByteArray();
```

Comment:
I'm not familiar with PBJ, but believe this will cause a new array allocation. If so, that would be very inefficient just to check for a certain character. Should use `getByte()` to check or see if there's an unsafe way to get access to the backing array.